roles/taca/templates/sthlm_fastq_taca_delivery.yml.j2

```diff
@@ -0,0 +1,21 @@
+log:
```

Comment:
upps_fastq_taca_delivery has never existed will not exist in the forseeable future.